### ISSUE TITLE
Help deploying the demo

### ISSUE THREAD

**Issue by paulmelnikow** [usdigitalresponse/neighbor-express#3]
HI! What a cool project! I wanted to ask for some help taking it for a spin! I'm working with some other folks in Brooklyn who are trying to organize a buyer's club and we're curious if Neighbor Express could be a good fit for what we're doing.

I've got the base copied to Airtable and the app forked and deployed to Now. I can fetch https://neighbor-express-nine.now.sh/api/get-cms and some data is coming back.

However this is what I'm seeing in the browser:

<img width="1160" alt="Screen Shot 2020-04-16 at 1 50 52 PM" src="https://user-images.githubusercontent.com/1487036/79489480-5b0e6900-7fe9-11ea-8d11-79710553d5e3.png">

Same result in Firefox, Safari, and Chrome.

There's a stack trace in the console but it's minified:

<img width="1014" alt="Screen Shot 2020-04-16 at 1 52 15 PM" src="https://user-images.githubusercontent.com/1487036/79489591-809b7280-7fe9-11ea-86d0-4642e0c8868a.png">

What's the best way to debug this?

Do you think the problem will manifest locally?

Is it possible to deploy the site with unminified sources?

Thanks!

**Comment by geoffreylitt** [comment]
Hi @paulmelnikow thanks for the note and sorry about the issue.

We had an error where our Airtable template and our readme were out of date -- things are changing fast over here! Could you please try starting over again, following the latest instructions in the Readme? I just tested that workflow and it should work.

Before you retry, you'll need to remove your existing airtable_base_id: download the [Zeit Now CLI](https://zeit.co/download) and do `now secrets remove airtable_base_id`. That will prompt zeit to ask you for a new base_id when you click the deploy button next.

The main change we made is that our Airtable template now has the correct schema, so when you make a copy of the base, it'll have the correct CMS fields in it.

The readme instructions for configuring Zeit were also slightly incorrect and are now fixed.

Let me know if you have any further issues/questions!

**Comment by keithk** [comment]
Really excited for you to try this out! I also created #4 to ensure we don't get this kind of failure in the future if the base CMS gets out of sync with the code, somehow.

**Comment by paulmelnikow** [comment]
Thanks you both! I'll try it now!

**Comment by paulmelnikow** [comment]
Hey, it works!

I had problems using the Zeit auto-clone and deploy workflow but I updated my repo, set the secrets in the CLI, and redeployed and it's working now!

https://neighbor-express-nine.now.sh/

**Comment by geoffreylitt** [comment]
Great, glad it's working!

Curious, any details on what went wrong w/ Zeit auto-clone?

I'm guessing it's due to the fact you did it twice, but would be nice to know in case there's a bug that would affect other people trying this out.

**Comment by paulmelnikow** [comment]
1. I think I got stuck in a loop. I had gotten it all the way deployed, then it asked if I wanted to link with GitHub. (I think I had disabled the GitHub integration on my Zeit account.) Once I linked GitHub and turned on the permissions it wanted, it seemed to want to create a second copy of the app. It did create a second repo.
2. The (first) repo it created did not have your commit history, only an "initial commit" which I imagine was a squashed commit.

This was all in the first pass (i.e. before I opened this issue). I think it's bugginess on Zeit's end.
